5_assignment/index.js

```diff
@@ -0,0 +1,35 @@
+var fs = require('fs');
+var filesArray = ["1.txt", "2.txt", "3.txt", "4.txt", "5.txt"];
+var fileCount = 0;
+
+function readFilePromise (fileName) {
+	return new Promise(function (resolve, reject) {
+		fs.readFile("./files_to_be_read/" + fileName, function(err, data) {
+			if(err) {
+				reject(err);
+			} else {
+			 	printPattern(fileName);
+				resolve(data);
```

Comment:
Argument is not used by the callback, you may as well not pass it if it's of no use!